modules/list_authors.rb

```diff
@@ -0,0 +1,12 @@
+module AuthorList
+  def list_all_authors
+    if @authors.length.zero?
+      puts 'No authors are available'
+    else
+      puts 'List of all Authors: '
+      @authors.each_with_index do |author, index|
+        puts "#{index}) [Author] First Name: #{author.publish_date}, Last Name: #{author.last_played_at} "
```

Comment:
- [ ] **Required:** Great job implementing the console interactivity. 👏 However, the wrong date format crashes the app. You need a way to safeguard the app from crashing by either
- request from the user to enter the correct date or 
- returning them back to a safe place in the app.